plugins/tech-insights-backend-module-jsonfc/src/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { Factory as JsonRulesEngineFactCheckerFactory } from './service/JsonRulesEngineFactChecker';
+export type { TechInsightCheckRegistry } from './service/CheckRegistry';
```

Comment:
This sounds like it should come from the tech insights package (rather than a module)? And just the implementation would live here?